setup.py

```diff
@@ -6,13 +6,14 @@
 
 setup(
     name="Open edX",
-    version="0.4",
+    version="0.5",
```

Comment:
Yes. I remember an email conversation `edx-platform: manual install required for new openedx directory` on 12/13/14 between @andy-armstrong and `Ahmad Abd Arrahman` when this `setup.py` was first added/modified and user's were getting exception `UserPartitionError: Unrecognized scheme random` since it didn't installed/updated automatically. This was the reason I added `Notes` section in description of this PR.
@feanil It would be great if it is installed automatically.